tests/unit/amo/components/TestAddon.js

```diff
@@ -680,6 +680,16 @@ describe(__filename, () => {
     expect(root.prop('location')).toEqual(location);
   });
 
+  it('does not show a ratings manager without a version', () => {
```

Comment:
Should we show a message or reason why we aren't displaying the ratings manager? I know this is an admin-only issue but as you mentioned on the call QA uses admin users a lot when doing manual testing... so they might run into a new error of "the ratings manager disappeared" ^_^

Maybe we should just show a message like "Ratings disabled because this add-on is incomplete" or something?

Reply:
Oh yeah, good idea. I updated the description with a screenshot of what it looks like now. I'm pretty sure no one would find a page like this in real life. It would be hard even for an admin to find one unless they knew to look for an incomplete add-on.

Reply:
And of course dev has an incomplete add-on featured on the home page for some reason 😛

Reply:
I dig it, thanks 😄

Reply:
Yeah I was gonna say -dev is always crazypants data.